### PR TITLE
Youth mpi

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

### CODE REVIEW

File: wazimap_za/templates/profile/profile_detail_youth.html [hunk 1]
```diff
@@ -187,9 +187,10 @@ <h2 class="header-for-columns"><a class="permalink" href="#mpi-score" id="mpi-sc
             <aside>
                 <p>* See <a href="http://opensaldru.uct.ac.za/bitstream/handle/11090/818/2016_169_Saldruwp.pdf?sequence=3" target="_blank">here</a> for details on the design and methodology of the Youth Multidimensional Poverty Index</p>
             </aside>
-            <div class="column-half">
-                {% include 'profile/_blocks/_stat_list.html' with stat=living_environment.youth_mpi_score stat_type='number' decimals="-3" %}
+            <div class="column-third">
```
Comment (longhotsummer) [inline wazimap_za/templates/profile/profile_detail_youth.html]: on medium displays this means the description can wrap. It takes up the whole line, it doesn't need to be 1/3rd does it?

File: wazimap_za/templates/profile/profile_detail_youth.html [hunk 2]
```diff
@@ -187,9 +187,10 @@ <h2 class="header-for-columns"><a class="permalink" href="#mpi-score" id="mpi-sc
...
+                {% include 'profile/_blocks/_stat_list.html' with stat=living_environment.youth_mpi_score_stat stat_type='number' decimals="-3" %}
```
Comment (longhotsummer) [inline wazimap_za/templates/profile/profile_detail_youth.html]: If you use living_environment.youth_mpi_score.youth_mpi_score for the stat, then you can use living_environment.youth_mpi_score for the chart I think.

So the stat just pulls out the one item from the chart's data.

File: wazimap_za/templates/profile/profile_detail_youth.html (hunk 1 above)
Comment (xybrnet) [inline wazimap_za/templates/profile/profile_detail_youth.html]: It is to make it align with the other stat numbers in this section. Should I rather change them all to only take up a quarter?

File: wazimap_za/templates/profile/profile_detail_youth.html (hunk 2 above)
Comment (xybrnet) [inline wazimap_za/templates/profile/profile_detail_youth.html]: 👍

File: wazimap_za/templates/profile/profile_detail_youth.html (hunk 1 above)
Comment (longhotsummer) [inline wazimap_za/templates/profile/profile_detail_youth.html]: I haven't checked the actual layout after you've moved things around. So as long as it makes sense visually and isn't unnecessarily constrained.

Comment (xybrnet) [inline wazimap_za/templates/profile/profile_detail_youth.html]: I don't think it does, as the rows in this section contain two donut charts which each take up a third too.

**Pull request merged**